sirepo/package_data/static/json/impactt-schema.json

```diff
@@ -92,73 +144,52 @@
             "zmu1": ["zmu1 [m]", "RPNValue", 0.0],
             "zmu2": ["zmu2", "RPNValue", 0.0]
         },
+        "impacttReport": {},
+        "simulationStatus": {},
+        "simulation": {
+            "elementPosition": ["Element Positioning", "ElementPosition", "absolute", "Beamline elements may be positioned relative to each other using Drifts for spacing, or positioned absolute using Elemedge."],
+            "visualizationBeamlineId": ["Beamline", "LatticeBeamlineList", ""]
+        },
         "_ELEMENT": {
             "name": ["NAME", "ValidatedString", "", ""],
             "l": ["L [m]", "RPNValue", 0, "The element length"],
-            "zedge": ["Z Edge", "RPNValue", 0, "Longitudinal position of the entrance face of the element"],
             "radius": ["Radius [m]", "RPNValue", 0],
             "x_offset": ["Misalignment X [m]", "RPNValue", 0],
             "y_offset": ["Misalignment Y [m]", "RPNValue", 0],
             "x_rotation": ["Rotation Error X [rad]", "RPNValue", 0],
             "y_rotation": ["Rotation Error Y [rad]", "RPNValue", 0],
             "z_rotation": ["Rotation Error Z [rad]", "RPNValue", 0]
         },
-        "DRIFTTUBE": {
-            "zedge": ["Z Edge", "RPNValue", 0, "Longitudinal position of the entrance face of the element"]
+        "BPM": {
```

Comment:
Remove the BPM element and replace with a set of elements which have a negative ele_type from here. Just the common items for now, it doesn't have to include all of them.
https://github.com/ChristopherMayes/lume-impact/blob/master/impact/parsers.py#L400

For example, different models for OFFSET_BEAM (name, l, x_offset, y_offset, py_offset, z_offset, pzoffset), WRITE_BEAM (name, l, filename, sample_frequency)